.github/workflows/ci.yml

```
@@ -9,7 +9,7 @@ on:
 jobs:
   cabal:
     name: cabal / ghc-${{matrix.ghc}} / ${{ matrix.os }}
-    continue-on-error: ${{ matrix.ghc == '9.2.2' }}
+    continue-on-error: ${{ matrix.ghc == '9.2.4' || matrix.ghc == '9.4.2'}}
```

I forget, this is so that we try to build on 9.2 and 9.4, but we don't guarantee that `password` works with those compilers, right?

Although it seems like 9.2 has been out for quite a while now, so we should probably just officially support it:

```suggestion
    # We don't guarantee that `password` works on the latest GHC.
    continue-on-error: ${{ matrix.ghc == '9.4.2' }}
```

---

Indeed. I'll add your suggestion.

I just added 9.4.2 so that when it's available, the jobs automatically try it (and not abort everything else if it's absent or fails)